arelle/plugin/validate/NL/rules/rules.py

```diff
@@ -0,0 +1,48 @@
+"""
+See COPYRIGHT.md for copyright information.
+"""
+from __future__ import annotations
+
+from typing import Any, Iterable
+
+from arelle import ModelDocument
+from arelle.ValidateXbrl import ValidateXbrl
+from arelle.typing import TypeGetText
+from arelle.utils.PluginHooks import ValidationHook
+from arelle.utils.validate.Decorator import validation
+from arelle.utils.validate.Validation import Validation
+from ..DisclosureSystems import (
+    DISCLOSURE_SYSTEM_NT16_PREVIEW,
+    DISCLOSURE_SYSTEM_NT17_PREVIEW,
+)
+from ..PluginValidationDataExtension import PluginValidationDataExtension
+
+_: TypeGetText
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16_PREVIEW,
+        DISCLOSURE_SYSTEM_NT17_PREVIEW,
+    ],
+)
+def rule_fr_kvk_1_01(
```

Comment:
Even though this is a pretty straight forward validation, it would be nice to have some unit tests for this.